third_party/ibis/ibis_oracle/__init__.py

```diff
@@ -25,6 +25,9 @@ class Backend(BaseAlchemyBackend):
     name = "oracle"
     compiler = OracleCompiler
 
+    def __init__(self, arraysize: int = 500):
```

Comment:
If we are overriding `__init__` should we add a `super()` call to the parent `__init__`?

Reply:
Yes good point -  I think we should to follow best-practice. I have added to latest PR